src/catalog/src/information_schema/tables.rs

```diff
@@ -44,6 +44,8 @@ impl InformationSchemaTables {
             ColumnSchema::new("table_schema", ConcreteDataType::string_datatype(), false),
             ColumnSchema::new("table_name", ConcreteDataType::string_datatype(), false),
             ColumnSchema::new("table_type", ConcreteDataType::string_datatype(), false),
+            ColumnSchema::new("table_id", ConcreteDataType::uint32_datatype(), true),
+            ColumnSchema::new("table_engine", ConcreteDataType::string_datatype(), true),
```

Comment:
MySQL has an `ENGINE` column in this table, I think we can reuse that column.
```suggestion
            ColumnSchema::new("engine", ConcreteDataType::string_datatype(), true),
```